ZeldaFullEditor/Data/DungeonsData.cs

```diff
@@ -36,5 +37,11 @@ public static class DungeonsData
         ///     A duplicate storage of all rooms to revert to when using the redo feature.
         /// </summary>
         public static List<Room>[] RedoRoom = new List<Room>[Constants.NumberOfRooms];
+
+        public static List<SpriteProperty> SpriteProperties = new List<SpriteProperty>();
+
+        public static byte[] SpriteDamageTaken = new byte[0x1000];
+
+        public static byte[] globalDamages = new byte[0x80];
```

Comment:
nit: camelCase when the file is PascalCase

Reply:
Done.